requirements.txt

```diff
@@ -6,7 +6,7 @@
 #
 asgiref==3.5.0
     # via django
-backports-zoneinfo==0.2.1
+# backports-zoneinfo==0.2.1
```

Comment:
@gsalunke do not commit changes in this file. This is only to get things working for your machine which does not have everything installed.